core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -36,7 +36,8 @@ public final class RuntimeConstants {
     }
   }
 
-  public static final String REVISION_SHORT = ProjectConstants.REVISION.substring(0, 8);
+  public static final String REVISION_SHORT = VERSION.length() > 8 ?
+      VERSION.substring(0, 8) : VERSION;
```

Comment:
revision instead of version.
```suggestion
  public static final String REVISION_SHORT = ProjectConstants.REVISION.length() > 8 ?
      ProjectConstants.REVISION.substring(0, 8) : ProjectConstants.REVISION;
```